src/controllers/itemDetails/index.js

```diff
@@ -1926,7 +1926,15 @@ export default function (view, params) {
     }
 
     function onPlayClick() {
-        playCurrentItem(this, this.getAttribute('data-action'));
+        let actionElem = this;
+        let action = actionElem.getAttribute('data-action');
+
+        if (!action) {
+            actionElem = actionElem.querySelector('[data-action]') || actionElem;
```

Comment:
The fallback to `actionElem` doesn't really make sense here since it would just end up with the action being undefined again.

Reply:
Originally it was:
```js
const actionElem = this.querySelector('[data-action]') || this;
const action = actionElem.getAttribute('data-action');
```

There are 2 layouts that we currently need to handle:
```
detailImageContainer <-- this
    poster data-action
```
and
```
btnPlay data-action <-- this
```

That worked fine. But I decide to also handle the item card with the overlay - I left the play button in Reactified poster:
```
poster data-action <-- this
    overlayPlay data-action
```

Alternatively, we can search for a parent with `data-action` starting from `e.target` using `dom.parentWithAttribute`.